query/QueryManager.java

```diff
@@ -50,17 +58,63 @@ public QueryManager(RPCTransaction rpcTransaction) {
 
     public Stream<ConceptMap> match(GraqlMatch query, GraknOptions options) {
         final QueryProto.Query.Req.Builder request = QueryProto.Query.Req.newBuilder().setMatchReq(
-                QueryProto.Graql.Match.Req.newBuilder().setQuery(query.toString()));
+                QueryProto.Query.Match.Req.newBuilder().setQuery(query.toString()));
         return iterateQuery(request, options, res -> res.getQueryRes().getMatchRes().getAnswersList().stream().map(ConceptMap::of));
     }
 
+    public QueryFuture<String> match(GraqlMatch.Aggregate query) {
```

Comment:
I know we serialize the number as a `String`, but that feels like an internal implementation detail. I don't think we should be returning it to the user as a `String`.

What data type does it have in Grakn Core? `double`?

Reply:
Yeah, I meant to ask you about how to return it to the user. Taking a look at how we're doing the conversion in Grakn Core makes sense.

Reply:
In Grakn Core it currently returns `Number`, which is either a `long` or a `double`. This is probably the best option as `Number` has the `longValue` and `doubleValue` methods.

Reply:
Addressed in https://github.com/graknlabs/client-java/pull/218/commits/7e6ebb80ca6044a40aba133db2f0a479c7b511c1